frontend/app/src/people/widgetViews/wantedView.tsx

```diff
@@ -555,21 +559,49 @@ export default function WantedView(props: any) {
                 }
               </div>
             </Pad>
-            <EuiText
+            <div
               style={{
-                fontSize: '14px',
-                color: '#8e969c',
-                fontWeight: '500'
+                display: 'flex',
+                flexDirection: 'row',
+                justifyContent: 'space-between',
+                alignItems: 'center'
               }}>
-              {estimate_session_length && 'Session:'}{' '}
-              <span
+              <EuiText
                 style={{
-                  fontWeight: '500',
-                  color: '#000'
+                  fontSize: '14px',
+                  color: '#8e969c',
+                  fontWeight: '500'
                 }}>
-                {estimate_session_length ?? ''}
-              </span>
-            </EuiText>
+                {estimate_session_length && 'Session:'}{' '}
+                <span
+                  style={{
+                    fontWeight: '500',
+                    color: '#000'
+                  }}>
+                  {estimate_session_length ?? ''}
+                </span>
+              </EuiText>
+              {ui?.meInfo?.isSuperAdmin && (
+                <EuiButtonIcon
+                  onClick={(e) => {
+                    e.stopPropagation();
+                    showModal();
+                    setDeletePayload({
+                      created: created,
+                      host: 'community.sphinx.chat',
```

Comment:
Instead of hard coding this bit I think we can use getHost() in `./src/hosts.ts`

Reply:
aah, silly mistake. Thanks for pointing this out.